tests/Microsoft.Identity.Test.LabInfrastructure/LabApiConstants.cs

```diff
@@ -39,9 +39,13 @@ public class LabApiConstants
         public const string True = "true";
         public const string False = "false";
 
-        public const string LabEndPoint = "https://msidlab.com/api/user";
+        public const string LabUserEndPoint = "https://msidlab.com/api/user/";
         public const string LabUserCredentialEndpoint = "https://msidlab.com/api/LabSecret";
         public const string LabAppEndpoint = "https://msidlab.com/api/app/";
         public const string LabInfoEndpoint = "https://msidlab.com/api/Lab/";
+
+        public const string LabOboPublicClientId = "be9b0186-7dfd-448a-a944-f771029105bf";
```

Comment:
we should not be hardcoding these app ids, can we use the `LabAppEndpoint` to query for these?

Reply:
I'm not sure. Just copied this from other tests.